authors/apps/articles/extra_methods.py

```diff
@@ -15,4 +16,18 @@ def create_slug(object_model, title):
     if qs_exists:
         new_slug = "{}-{}".format(slug, random_string_generator(size=4))
         return create_slug(title, new_slug)
-    return slug
\ No newline at end of file
+    return slug
+
+def likes_count(article):
+    likes = ArticleLike.objects.filter(article=article['id'], like_value=True)
+    return len(likes)
+
+def dislikes_count(article):
+    dislikes = ArticleLike.objects.filter(article=article['id'], like_value=False)
+    return len(dislikes)
+
```

Comment:
This is good @AtamaZack. However, like count and dislike count can be done by one method. I think if you passed that method the article and like value, it would combine the two methods into one.

Reply:
Well done, @AtamaZack, I like the thought process that you have applied to your Feature. Amazing Feature. World class.  It good you have also corrected the typo in the response. 👌